.github/workflows/ci.yaml

```diff
@@ -66,7 +63,7 @@ jobs:
   "lint_fmt":
     name: lint:fmt
     runs-on: ubuntu-latest
-    if: needs.changes.outputs.crates != '[]' && !github.event.pull_request.draft
```

Comment:
I want to be able to test when I am in draft whether it is ready for a review. Right now I have to test that locally.

Reply:
It depends on how we want to use `DRAFT`. 

I used `DRAFT`, when I explicitly wanted to push code while not triggering the CI pipeline.
Once ready the code was ready for the CI pipeline, I removed the `DRAFT` to test the CI pipeline 
Once ready for the reviewers, I added the `reviewers`.

Reply:
When something is not in draft, then it should be ready for review imo.

Reply:
I do not see the above mentioned workflow being worse by running CI.

Reply:
Problem is I keep having to remove these contraints temporarily in order for me to test something without undrafting the PR which annoys me.

Reply:
Since it is already possible to run the CI pipelines (by not adding the `DRAFT` label), I don't see a reason to change the behavior.